inc/saiqueue.h

```diff
@@ -311,6 +317,12 @@ typedef enum _sai_queue_stat_t
     /** Get/set WRED red marked bytes count [uint64_t] */
     SAI_QUEUE_STAT_WRED_ECN_MARKED_BYTES = 0x00000023,
 
+    /** Get current queue occupancy percentage [uint64_t] */
+    SAI_QUEUE_STAT_CURR_OCCUPANCY_LEVEL = 0x00000024,
+
```

Comment:
we already have SAI_QUEUE_STAT_CURR_OCCUPANCY_BYTES  and SAI_QUEUE_STAT_WATERMARK_BYTES . Why we need this ? is it related to voq changes or general changes ?

Reply:
DNX Fabric ports do not support queue stats in byte units.  Fabric port queue stats use general units for queue statistics.  This can also be used in general cases.  Percentage is a generic representation of queue occupancy and watermark that abstracts port speed and configured queue sizes.